azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -40,6 +40,10 @@ jobs:
     clean: true
     submodules: recursive
     persistCredentials: True
+  - bash: |
```

Comment:
this is duplicated, right? we set credentials in automation-cert.yml

Reply:
For whatever reason the state is not carried over, i think it's something that happens in pipeline when templates are used...

I tried it here, failed https://identitydivision.visualstudio.com/Engineering/_build/results?buildId=1570162&view=results